lib/main_common.pm

```diff
@@ -858,7 +858,8 @@ sub load_inst_tests {
     if (get_var('ENCRYPT_CANCEL_EXISTING') || get_var('ENCRYPT_ACTIVATE_EXISTING')) {
         loadtest "installation/encrypted_volume_activation";
     }
-    if (get_var('MULTIPATH') or get_var('MULTIPATH_CONFIRM')) {
+    # SLE-15 SP2 pushes the multipath activation dialog after the scc_registration dialog, see below
+    if ((get_var('MULTIPATH') or get_var('MULTIPATH_CONFIRM')) and not is_sle('15-SP2+')) {
```

Comment:
Instead of negation here you can simply write `('<15-SP2')`

Reply:
The idea of this formulation is to forestall the concern that the `loadtest "installation/multipath"` invocation might have been made more restricted than before. The conditions `not is_sle('15-SP2+')` and later `if (is_sle('15-SP2+') ...` are clearly complementary, so this is obviously not the case. Your proposal `('<15-SP2')` is equivalent, but just slightly less obvious IMHO -- a matter of taste. What do you think?

Reply:
It's indeed matter of taste, so we can leave it as it is. Just wanted to be sure that you are aware of that </> syntax.